R/helpers.R

```diff
@@ -306,6 +306,13 @@ LoadH5AD <- function(path) {
   IsMetaData <- function(md) {
     return(Exists(name = md) && inherits(x = adata[[md]], what = 'H5Group'))
   }
+  LoadFromDataSet <- function(md) {
+    if (Exists(name = md) && inherits(x = adata[[md]], what = 'H5D') && 'index' %in% names(adata[[md]][])) {
+      return (adata[[md]][]$index)
```

Comment:
This is two separate loadings of `adata[[md]]`

Reply:
good point - I'll fix it in 0.4.5